client/src/components/TypeList.vue

```diff
@@ -26,24 +27,33 @@ export default Vue.extend({
       selectedType: '',
       editingType: '',
       editingColor: '',
+      editingThickness: 5,
+      editingFill: false,
+      editingOpacity: 1.0,
```

Comment:
Form validation is needed, even though you have a rule on your editingThicknes, if you set it below 0, it still works.

You can wrap all the form elements in a `<v-form v-model="valid">` and the state of valid will track with all sub-inputs.

Reply:
I have no problem with you opening a PR if you already did the changes that you feel are needed.  Now that I look at it I don't know what I was doing with that slider and why I felt I needed an input.  I think it was just because I had an input at first before adding the slider.